apps/web/src/server/app/auth/auth.service.js

```diff
@@ -49,10 +55,23 @@ export const acquireTokenSilent = async (account) => {
  * @param {AccountInfo} account
  * @returns {Promise<void>}
  */
-export const clearCacheForAccount = async (account) => {
+const clearCacheForAccount = async (account) => {
 	await msalClient.getTokenCache().removeAccount(account);
 };
 
+/**
+ * Completely Wipe out Session and Account Data
+ *
+ * @param {SessionWithAuth} session
+ * @param {any} response //TODO find response type
```

Comment:
you can try:
`@param {import('express').Response} response`

Reply:
done